packages/terra-pills/src/FilterPills.jsx

```diff
@@ -0,0 +1,448 @@
+      } else {
+        setTabIndex('-1');
+        focusNode.current = pills.length;
+      }
+    } else {
+      setShowRollupPillInteraction(false);
+    }
+    setIsCollapsed(!isCollapsed);
+  };
+
+  const handlePillListOnblur = () => setContainerTabindex('-1');
+
+  const filterPillsProps = {};
+  filterPillsProps.onKeyDown = handlePillListKeyDown;
+  filterPillsProps.onBlur = handlePillListOnblur;
+
+  const pillListClassNames = classNames(
+    cx([
+      'pills-group',
+      theme.className,
+    ]),
+    customProps.className,
+  );
+
+  const pillInteractionHintID = 'terra-pills-interaction-hint';
+  const pillGroupInteractionHintID = 'terra-pills-group-interaction-hint';
+  let removedPillInteractionHint = '';
+  const pillGroupAriaDescribedBy = ariaDescribedBy ? `${ariaDescribedBy} ${pillGroupInteractionHintID}` : pillGroupInteractionHintID;
+  let pillGroupInteractionHint = intl.formatMessage({ id: 'Terra.pills.hint.pillList' }, { numberOfPills: React.Children.count(children) });
+  if (isCollapsible && (rollUpCount > 0) && isCollapsed) {
+    pillGroupInteractionHint += `, ${intl.formatMessage({ id: 'Terra.pills.hint.rollupNotVisible' }, { pillsNotVisibleCount: rollUpCount })}`;
+  } else if (isCollapsible && !isCollapsed && showRollupPillInteraction) {
+    removedPillInteractionHint = 'hidden items are displayed now...'; // needs transtaions lp052179
+  }
+  if (pillRemoved) {
+    removedPillInteractionHint = `${removedLabel.current} was removed...`; // needs translations lp052179
+  }
+
+  const renderChildren = (items) => {
+    const pills = React.Children.map(items, (pill) => {
+      if (React.isValidElement(pill)) {
+        return React.cloneElement(pill, {
+          onRemove: handleOnRemove, onSelect: handleOnPillSelect, pillInteractionHintID,
+        });
+      }
+      return undefined;
+    });
+
+    const reducedArray = pills.splice(0, updatedCount);
+
+    return reducedArray;
+  };
+
+  return (
+    <div
+      {...customProps}
+      {...filterPillsProps}
+      aria-label={!ariaLabelledBy ? ariaLabel : undefined}
+      aria-labelledby={ariaLabelledBy}
+      aria-describedby={pillGroupAriaDescribedBy}
+      className={pillListClassNames}
+      ref={filterPillsRef}
+      role="list"
+      tabIndex={containerTabindex}
+    >
+      <VisuallyHiddenText
+        aria-live="assertive"
+        id={pillInteractionHintID}
+        text={removedPillInteractionHint}
+        aria-hidden="true"
+      />
+      <VisuallyHiddenText
+        aria-live="assertive"
+        id={pillGroupInteractionHintID}
+        text={pillGroupInteractionHint}
+        aria-hidden="true"
+      />
+      {children ? renderChildren(children) : []}
+      {(isCollapsible && rollUpCount > 0) && (
+        <RollUpPill
+          isCollapsed={isCollapsed}
+          onSelectRollUp={handleOnSelectRollUp}
+          rollupCount={rollUpCount}
+        />
+      )}
+    </div>
+  );
+};
+
+FilterPills.Pill = Pill;
```

Comment:
With our newer components we've moved away from using the dot notation for subcomponent. You can make additional exports like this: https://github.com/cerner/terra-framework/blob/main/packages/terra-infinite-list/src/index.js#L5-#L9

Reply:
changed ae07eb46c1332427df41f2b68be6f9007c75930c